src/Sulu/Bundle/AdminBundle/Resources/js/components/Matrix/tests/Matrix.test.js

```diff
@@ -162,7 +167,9 @@ test('Changing a value should call onChange ', () => {
         },
     };
 
-    matrix.find(Item).at(3).simulate('click');
+    // eslint-disable-next-line testing-library/no-node-access
+    const item = screen.queryAllByLabelText('su-pen')[1].parentElement;
+    fireEvent.click(item);
```

Comment:
Is fine at me at current state as we do not have good accessibilty for this component yet we need implement hack via parentElement here.